code/modules/mod/mod_control.dm

```diff
@@ -278,6 +278,16 @@
 			balloon_alert(wearer, "retract parts first!")
 			playsound(src, 'sound/machines/scanbuzz.ogg', 25, FALSE, SILENCED_SOUND_EXTRARANGE)
 			return
+
+	// SKYRAT EDIT ADDITION START - Can't remove your MODsuit from your back when it's still active (as it can cause runtimes and even the MODsuit control unit to delete itself)
+	if(!active)
```

Comment:
I don't know how it was functional before, but now it should be functional again.
```suggestion
	if(active)
```